### PR TITLE
Inline Inputs linter for Columns work

### PR DESCRIPTION
**Pull request by SeanArmstrong**
This can be merged into master after #1304 has been merged

### CODE REVIEW

**Comment by SeanArmstrong** [comment]
This requires a few more changes
```
   9:14  error  'label' is missing in props validation                react/prop-types
  11:28  error  'htmlFor' is missing in props validation              react/prop-types
  12:15  error  'label' is missing in props validation                react/prop-types
  26:21  error  Do not use Array index in keys                        react/no-array-index-key
  67:10  error  'label' PropType is defined but prop is never used    react/no-unused-prop-types
  75:12  error  'htmlFor' PropType is defined but prop is never used  react/no-unused-prop-types
  75:28  error  Unexpected trailing comma                             comma-dangle
```

**Comment by SeanArmstrong** [comment]
Require re-reviewing

File: src/components/inline-inputs/inline-inputs.js [hunk 1]
```diff
@@ -1,35 +1,29 @@
...
-let columnWrapper = (children) => {
+const columnWrapper = (children) => {
   let inputs = children;
 
   if (!Array.isArray(inputs)) {
     inputs = [children];
   }
 
   return inputs.map((input, index) => {
+    /* eslint-disable react/no-array-index-key */
```
Comment (abramin) [inline src/components/inline-inputs/inline-inputs.js]: is there no way to satisfy the rule? using something in the input for example?
It's a good rule 😢

File: src/components/inline-inputs/inline-inputs.js [hunk 2]
```diff
@@ -48,12 +53,43 @@ InlineInputs.propTypes = {
...
+   */
+  htmlFor: PropTypes.string
+};
+
+Label.defaultProps = {
+  label: '',
+  htmlFor: null
+};
+
+
```
Comment (abramin) [inline src/components/inline-inputs/inline-inputs.js]: extra space

File: src/components/inline-inputs/inline-inputs.js (hunk 1 above)
Comment (SeanArmstrong) [inline src/components/inline-inputs/inline-inputs.js]: It is but the only way i can see to fix it is to expect a unique identifier on each of the inputs which is a breaking change. We could maybe look to address all these for version 2.0?

**Comment by abramin** [comment]
Tested in IE 11, Firefox, Edge, Chrome, Safari - looks ok in Demo site, no console errors.

**Pull request merged**